datepicker-directive.js

```diff
@@ -125,6 +125,8 @@ function datepickerFactory($filter, $templateCache, $timeout) {
         }
         if(ctrl.options.modelType === 'string') {
           ctrl.model = dateFilter(ctrl.date, ctrl.options.format);
+        } if(ctrl.options.modelType === 'w3cDate') {
+          ctrl.model = util.w3cDate(ctrl.date);
```

Comment:
Doesn't using `modelType = 'string'` already do the same thing? You just need to set the appropriate format? If not, maybe we just need a `modelFormat` option? Adding `w3cDate` seems really specific.